.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,94 @@
+name: Bug Report
+description: File a bug report
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        Note: Please search to see if an issue already exists for the bug you encountered.
+  - type: textarea
+    id: current-behavior
+    attributes:
+      label: Current Behavior
+      description: A concise description of what you're experiencing.
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: A concise description of what you expected to happen.
+      placeholder: Tell us what you want to see!
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to Reproduce
+      description: Detailed steps to reproduce the behavior.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: What version of reneterd are you running? If you are running from source, please provide the commit hash.
+      placeholder: v1.0.0
+    validations:
+      required: true
+  - type: input
+    id: os
+    attributes:
+      label: What operating system did the problem occur on (e.g. Ubuntu 22.04, macOS 12.0, Windows 11)?
+    validations:
+      required: true
+  - type: input
+    id: apCfg
+    attributes:
+      label: Autopilot Config
+      description: |
+        The configuration of your autopilot
+        ```bash
+        curl -u ":[YOUR_PASSWORD]" http://localhost:9980/api/autopilot/config
+        ```
+      placeholder: Paste the output of the above command here ```
+    validations:
+      required: true
+  - type: input
+    id: busCfg
+    attributes:
+      label: Bus Config
+      description: |
+        The configuration of your bus
+        ```bash
+        curl -u ":[YOUR_PASSWORD]" http://localhost:9980/api/bus/setting/redundancy
+
+        curl -u ":[YOUR_PASSWORD]" http://localhost:9980/api/bus/setting/gouging
```

Comment:
we have these settings:
```go
	SettingContractSet      = "contractset"
	SettingGouging          = "gouging"
	SettingRedundancy       = "redundancy"
	SettingS3Authentication = "s3authentication"
	SettingUploadPacking    = "uploadpacking"
```
we should probably ask for every single one except for S3 auth?